aws/validators.go

```diff
@@ -2133,3 +2133,15 @@ func validateGameliftOperatingSystem(v interface{}, k string) (ws []string, erro
 	}
 	return
 }
+
+func validateGuardDutyIpsetFormat(v interface{}, k string) (ws []string, errors []error) {
+	value := v.(string)
+	validType := []string{"TXT", "STIX", "OTX_CSV", "ALIEN_VAULT", "PROOF_POINT", "FIRE_EYE"}
```

Comment:
Nitpick: Can you use the SDK provided constants please? e.g. `guardduty.IpSetFormatAlienVault`